telemetry/definitions/commonDefinitions.json

```diff
@@ -3882,10 +3926,10 @@
         {
             "name": "codeTransform_logGeneralError",
             "passive": true,
-            "description": "Deprecated - avoid logging every service call",
+            "description": "A general error has occurred.",
```

Comment:
We can keep this (remove the deprecated comment)

Reply:
Okay, we've decided to keep this metric as being used and updated the document describing each metric type.

Reply:
Can you combine this metric to cover both success and failure case. I'm sure why we need a dedicated metric to cover failure

Reply:
There are errors that are either emitted by the API or that occur locally (e.g., build errors) that we want to be emitted so that we can track them for our operations.